libs/blocks/hero-marquee/hero-marquee.css

```diff
@@ -405,10 +405,6 @@ html[dir="rtl"] .hero-marquee li.icon-item span.icon {
     flex-direction: row;
   }
 
-  .hero-marquee.media-cover picture {
```

Comment:
I'm a bit weary to change behavior potentially used on production, but I'm not noticing any change when removing this. Did you investigate any additional impact this might have?